DeusEx/Classes/DXRReduceItems.uc

```diff
@@ -23,17 +122,25 @@ function ReduceAmmo(float mult)
 
     foreach AllActors(class'Weapon', w)
     {
-        if( w.PickupAmmoCount > 0 )
-            w.PickupAmmoCount = Clamp(float(w.PickupAmmoCount) * mult, 1, 1000);
+        if( w.AmmoName != type && ClassIsChildOf(w.AmmoName, type) == false ) continue;
+        if( w.PickupAmmoCount > 0 ) {
+            i = Clamp(float(w.PickupAmmoCount) * mult, 1, 1000);
+            l("reducing ammo in "$ActorToString(w)$" from "$w.PickupAmmoCount$" down to "$i);
+            w.PickupAmmoCount = i;
+        }
     }
 
     foreach AllActors(class'Ammo', a)
     {
-        if( a.AmmoAmount > 0 && ( ! CarriedItem(a) ) )
-            a.AmmoAmount = Clamp(float(a.AmmoAmount) * mult, 1, 1000);
+        if( ! a.IsA(type.name) ) continue;
+        if( a.AmmoAmount > 0 && ( ! CarriedItem(a) ) ) {
+            i = Clamp(float(a.AmmoAmount) * mult, 1, 1000);
+            l("reducing ammo in "$ActorToString(a)$" from "$a.AmmoAmount$" down to "$i);
+            a.AmmoAmount = i;
+        }
     }
 
-    ReduceSpawnsInContainers(class'Ammo', int(mult*100.0) );
+    ReduceSpawnsInContainers(type, int(mult*100.0) );
 }
 
 function ReduceSpawns(class<Actor> classname, int percent)
```

Comment:
maybe if percent > 100%, we could make it increase the NumCopies, so you could do like 500% multitools